src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -939,8 +939,20 @@ else if (Endpoint.SELECTED_ENDPOINTS_PROPERTY_NAME
      */
     void endpointExpired(AbstractEndpoint endpoint)
     {
-        if (endpoints.remove(endpoint.getID()) != null)
+        final AbstractEndpoint removedEndpoint;
+        synchronized (endpoints)
```

Comment:
`endpoints` is already a `ConcurrentHashMap`, so i don't think the synchronized is needed here?

Reply:
I also think it is not necessary, but it is wrapped into `syncronized` in other places, so I thought it's probably missed here :\ Do you think it is not necessary here actually and I can remove it?

Reply:
hm, i see it's synchronized in some places to allow a broader operation to be atomic with the modification of the map.  yes, i think it should be synchronized here and, after looking, i see at least one bug elsewhere due to operations happening outside of a synchronized block.